docs/about.md

```diff
@@ -34,3 +34,22 @@ Talawa’s desired main features are:
 1. Donation acceptance
 1. Service provider level multi-organization capability
 1. Plugin support for administration services such as billing
+
+### Technologies Used
+
+A brief introduction to all of the Technologies used by Talawa and the Github repositories of different projects. This serves as a resource to newcomers seeking a perspective of the collective community efforts. Link to the four major repositories under Talawa are given below:
+
+* [Talawa](https://github.com/PalisadoesFoundation/talawa): HTML, XML, YAML, SVG, Gradle, Python, JSON, Flutter and Dart are the technologies that are used by talawa.
+* [Talawa-API](https://github.com/PalisadoesFoundation/talawa-api): Talawa-API uses TypeScript, JSON, YAML, Shell and Python technologies.
+* [Talawa-Admin](https://github.com/PalisadoesFoundation/talawa-admin): Talawa-Admin uses CSS, TypeScript, TSX, YAML, Shell JSON, GraphQL and JavaScript technologies.
+* [Talawa-Docs](https://github.com/PalisadoesFoundation/talawa-docs): Talawa-docs uses technologies like HTML, CSS, YAML, SVG, TSX, SCSS, JSON and JavaScript.
+
+Below is the list of various technologies used:
+
+1. Front-end Technologies: HTML, CSS, and JavaScript (React) is used to create the website's front-end user interface. And, Flutter is used to create the mobile application.
+2. Back-end Technologies: Talawa uses server-side scripting languages like GraphQL and TypeScript to handle the website's back-end functionality.
+3. Database Management System: Talawa API uses database management system like MongoDB with the mongoose ORM (Object–relational mapping) which is used to store and manage the website's data.
+4. Security: Talawa uses various encryption technologies to protect sensitive data like SSL (Secure Socket Layer) which helps to create a secure encrypted connection between the server (our project) and the client, and protects Talawa from hacking and other security threats.
```

Comment:
Remove `like` replace with `including`. Mention that adding SSL was a GSoC 2023 idea.